pkg/connection/connection.go

```diff
@@ -71,9 +66,14 @@ func CheckAuthFromContext(ctx context.Context, runPath string) error {
 
 // Dial creates a gRPC connection to the server.
 func Dial(ctx context.Context, address, passwordFile string) (proto.ManageClient, func() error, error) {
-	pw, err := os.ReadFile(passwordFile)
-	if err != nil {
-		return nil, nil, fmt.Errorf("reading manage auth password file %q: %w", passwordFile, err)
+	pw := []byte(shared.DevelopmentPassword)
+	dev, _ := strconv.ParseBool(os.Getenv("OPENSLIDES_DEVELOPMENT"))
+	if !dev {
+		var err error
+		pw, err = os.ReadFile(passwordFile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("reading manage auth password file %q: %w", passwordFile, err)
+		}
```

Comment:
Only style: I would only write the pw variable in the success path.

```go
filePW, err := os.ReadFile(passwordFile)
if err != nil {
  return nil, ...
}
pw = filePW
```

Since you return in the error case, the result is the same. But if in the future, someone decides, that an error here should only be a logged warning and changes your code to:

```go
	       var err error
		pw, err = os.ReadFile(passwordFile)
		if err != nil {
			log.Printf("reading manage auth password file %q: %v", passwordFile, err)
		}
```

Then the variable pw will be rewritten by accident.